test/functional/localConfig.json

```diff
@@ -2,7 +2,7 @@
     "desktop": {
       "testsFolder": "test/functional/specs/desktop/",
       "browser": [
-        "chrome", "safari"
```

Comment:
Are we disabling Safari for this particular test?

Reply:
No, I din't mean to check this change in.